src/index.ts

```diff
@@ -107,151 +108,106 @@ export interface CreateGraphOptionsSpec {
   edgeTable: string;
 }
 
-function fileToText(file: File): Promise<string> {
-  return new Promise((resolve, reject) => {
-    const reader = new FileReader();
-    reader.onload = (e) => {
-      if (e.target === null || typeof e.target.result !== 'string') {
-        throw new Error();
-      }
-      resolve(e.target.result);
-    };
-    reader.onerror = (e) => {
-      reject();
-    };
-
-    reader.readAsText(file);
-  });
-}
-
 class MultinetAPI {
-  public client: Client;
+  public axios: MultinetAxiosInstance;
 
   constructor(baseURL: string) {
-    this.client = new Client(baseURL);
+    this.axios = multinetAxiosInstance(axios.create({
+      baseURL,
+    }));
```

Comment:
To mirror my suggestion above:
```suggestion
    this.axios = multinetAxiosInstance({ baseURL });
```

Reply:
Shoot, I forgot to use the "add suggestion to batch" feature for this and the previous commit 🤕